### PR TITLE
Fix/cannot update dao listing while rendering link dao

### PR DESCRIPTION
**Pull request by mquellhorst**
State was set inside a component which is rendered inside another component.
This makes it impossible to update the parent component and results in an error message in the console.
See:
https://github.com/facebook/react/issues/18178

### CODE REVIEW

**Comment by dan13ram** [comment]
Hey @mquellhorst I've made some slightly different changes to show the daoHaus urls instead of etherscan along with fixing that bug in #204

Can you check https://github.com/MetaFam/TheGame/blob/d3faafe5416984fc6ed210ba461809d6e188892e/packages/web/components/Player/Section/PlayerMemberships.tsx ?

**Comment by dan13ram** [comment]
Maybe we will push this first and then I'll rebase my branch upon this. What say? @mquellhorst

File: packages/web/components/Player/Section/PlayerMemberships.tsx [hunk 1]
```diff
@@ -90,17 +79,27 @@ const DaoListing: React.FC<DaoListingProps> = ({
...
     message = `XP: ${Math.floor(memberXp || 0)}`;
   } else {
     message = `Shares: ${memberShares}/${daoShares}`;
   }
 
-  const [isLink, setIsLink] = useState(false);
-
   return (
-    <LinkDao chain={chain || ''} address={address || ''} setIsLink={setIsLink}>
+    <LinkDao
```
Comment (dan13ram) [inline packages/web/components/Player/Section/PlayerMemberships.tsx:98]: chain and address are no longer necessary for this component, maybe we can remove it?

**Comment by mquellhorst** [comment]
I see, yeah sounds good!

File: packages/web/components/Player/Section/PlayerMemberships.tsx (hunk 1 above)
Comment (mquellhorst) [reply packages/web/components/Player/Section/PlayerMemberships.tsx:98]: You're right, I'll remove them then we can merge this so you can rebase and merge yours

**Review by dan13ram — APPROVED** [review]

**Pull request merged**